collection/collection.go

```diff
@@ -68,13 +68,12 @@ func (sm *SetMaintainer) subscribe(bus event.Bus) {
 		event.ETRemoteDatasetIssueClosed,
 
 		// automation events
-		event.ETAutomationWorkflowStarted,
-		event.ETAutomationWorkflowStopped,
```

Comment:
Should we also remove these from the events package? Seems like we can/should remove them entirely

Reply:
I was thinking we hold off on removing events until frontend is more settled (basically when our most recent list is burned down). Then do a review on what events frontend is using/not using, and removing anything that is erroneous.